src/content/en/progressive-web-apps/checklist.md

```diff
@@ -310,13 +310,13 @@ and [social discovery](/web/fundamentals/discovery-and-monetization/social-disco
             Recipe type markup</a> for Rich Cards.
           </li>
           <li>
-            A news app you should have the <a href="/search/docs/data-types/articles">
+            A news app should have the <a href="/search/docs/data-types/articles">
             NewsArticle type markup</a> for Rich Cards and/or 
             <a href="https://www.ampproject.org/docs/reference/spec.html">
             AMP support</a>.
           </li>
           <li>
-            An ecommerce app should have the
+            An e-commerce app should have the
```

Comment:
Citation needed?

http://english.stackexchange.com/questions/174165/correct-spelling-and-or-hyphenation-for-electronic-commerce

https://www.merriam-webster.com/dictionary/ecommerce